src/tray/cocoa/SDL_tray.m

```diff
@@ -276,13 +276,22 @@ void SDL_SetTrayTooltip(SDL_Tray *tray, const char *tooltip)
     return entry->submenu;
 }
 
-const SDL_TrayEntry **SDL_GetTrayEntries(SDL_TrayMenu *menu, int *size)
+SDL_TrayEntry **SDL_GetTrayEntries(SDL_TrayMenu *menu, int *size)
 {
+    SDL_TrayEntry **result = (SDL_TrayEntry **)SDL_malloc(sizeof((menu->nEntries + 1) * sizeof(*result));
```

Comment:
```suggestion
    SDL_TrayEntry **result = (SDL_TrayEntry **)SDL_malloc((menu->nEntries + 1) * sizeof(*result));
```

Reply:
Oops, sorry about that. I had also forgotten to change `out` for `result`. They should both be fixed in all files now, but I'll keep an eye on the CI.